### ISSUE TITLE
Consider stabilization

### ISSUE THREAD

**Issue by Ekleog** [alexcrichton/scoped-tls#5]
This crate has seen 1.2M download, yet its (stable, I'm assuming the `nightly` feature counts as unstable) API appears to have not seen a single change in two years, nor has any issue been raised with it that could impact stability of its API.

Just thinking maybe this would make `scoped-tls` a good candidate for stabilization? :)

-----

Concerns:
- [ ] `:vis` is not stable, and required to merge the feature `nightly` into the default behaviour

**Comment by alexcrichton** [comment]
I think this'd be a great candidate for stabilization! I think that the nightly feature also is likely no longer needed, I believe the relevant feature was stabilized in rustc.

If you'd like to help out I'd love to have a PR to bump the versions and remove the nightly feature, and after that I think it's safe to release 1.0!

**Comment by Ekleog** [comment]
Hmm, it looks like `:vis` is still feature-gated on play.rust-lang [(example)](http://play.rust-lang.org/?gist=2cca6d10837291758ddae41122b32eb5&version=nightly&mode=debug&edition=2015).

As it [looks like](https://github.com/rust-lang/rust/issues/41022#issuecomment-410608447) it is going to be stabilized in the next release, would you rather wait for the next release, or would it make sense to switch to 1.0 with the `nightly` feature not included in stability guarantees (afaik the only way to handle partially-stable APIs *à la* rustc) and when `:vis` is merged, merge the two features? (as it would be a strict extension of the syntax, I don't think it'd break backwards-compat).

The safest option is probably to wait a bit more, though, but it's your pick :)

**Comment by alexcrichton** [comment]
Ah oops thanks for checking! In that case yeah let's wait for the next release, and then 1.0 sounds good to me!

**Comment by Xanewok** [comment]
Heads up - `:vis` is stable since 1.30 and I submitted a PR that removes the feature flag altogether: #10
